### DISCUSSION TITLE
Frankfurt am Main (Germany) is ready

### DISCUSSION THREAD

**Discussion by BerenikeRe** [EqualStreetNames/equalstreetnames#766] [New city]
Hey everyone!
After one year of working on it I finally finished all the Wikidata and most of Wikipedia entries of Frankfurt am Main, Germany.
https://github.com/BerenikeRe/equalstreetnames-frankfurt-am-main (EDIT: changed the link)
I would like to transfer the repository to the EqualStreetNames organization. A mapbox token is integrated.
Best wishes from Mainz
Berenike

**Comment by jbelien** [comment]
Awesome! 🎉 

You're already a member of EqualStreetNames organization so you should already be able to transfer the new repository.

I'll take care of the next steps in the next few days.

**Comment by BerenikeRe** [comment]
Done! (hopefully 🤞🏻)
Thank you very much!

**Comment by jbelien** [comment]
🚀 Deployed: https://frankfurt.equalstreetnames.eu/

@BerenikeRe Absolutely amazing work! 🤩 Thanks a lot!
